master.yaml

```diff
@@ -38,6 +38,17 @@ mysql:
     MYSQL_ROOT_PASSWORD: root
   volumes:
     - ./volumes/mysql:/docker-entrypoint-initdb.d/:ro
+    
+mongodb:
+  image: 'mongo'
+  container_name: 'mongo-container'
+  environment:
+    - MONGO_INITDB_DATABASE='stackbox'
+  volumes:
+    - ./volumes/mongo/init-mongo.js:/docker-entrypoint-initdb.d/init-mongo.js:ro
+    - ./mongo-volume:/data/db
+  ports:
+    - '27017-27019:27017-27019'
```

Comment:
This means range of ports right? We wont be able to write generic connectivity from backend in this case 😢 Is it possible to change to run on a single port?

Reply:
I have changed the port in a new pull request. But, I don't have edit access to wiki